Source/WebView/turbo.js

```diff
@@ -1,19 +1,18 @@
 (() => {
+  const TURBO_LOAD_TIMEOUT = 4000
+
   // Bridge between Turbo JS and native code. Built for Turbo 7
   // with backwards compatibility for Turbolinks 5
   class TurboNative {
     constructor() {
       this.messageHandler = webkit.messageHandlers.turbo
-      this.registerAdapter()
      }
 
     registerAdapter() {
       if (window.Turbo) {
         Turbo.registerAdapter(this)
       } else if (window.Turbolinks) {
         Turbolinks.controller.adapter = this
-      } else {
-        this.pageLoadFailed()
```

Comment:
I'm wary of failing silently. What do you think about throwing an error if we can't register the adapter?

Reply:
Makes perfect sense 👍  https://github.com/hotwired/turbo-ios/pull/52/commits/d6dfae268d6fc6241f2dd2a6e34209bbc0b49da0